src/borrowing/interior-mutability.md

```diff
@@ -2,73 +2,59 @@
 minutes: 10
 ---
 
-<!-- NOTES:
-Introduce the concept, with an example based on Mutex showing an `&self` method doing mutation; reference Cell/RefCell without detail.
--->
-
 # Interior Mutability
 
-Rust provides a few safe means of modifying a value given only a shared
-reference to that value. All of these replace compile-time checks with runtime
-checks.
-
-## `Cell` and `RefCell`
+In some situations, exclusive (`mut`) access cannot be represented at compile
```

Comment:
Another thought I had while reading this is that "shared reference" and "exclusive reference" don't express the read-only and mutable access well. Could we change the slide to use "read-only reference" and "mutable reference" here instead?

Reply:
We use those terms for references earlier in the course (https://google.github.io/comprehensive-rust/references/shared.html) and I think they are the preferred term in the Rust community. I can replace the parenthesized `(mut)` with `(mutable)` if that helps.

Reply:
I honestly don't know what the canonical term is... but I tried looking it up:

- The Rust Book talks about _immutable_ and _mutable references_ in [References and Borrowing](https://doc.rust-lang.org/book/ch04-02-references-and-borrowing.html#references-and-borrowing).
- The Rust Reference mentions _shared borrow_ and _mutable borrow_ in [Borrow operators](https://doc.rust-lang.org/stable/reference/expressions/operator-expr.html#borrow-operators).